### ISSUE TITLE
BOM in combined filed bridge.js, bridge.min.js, missing version header, missing comments

### ISSUE THREAD

**Issue by txdv** [bridgedotnet/Bridge#288]
If I compile the solution Bridge.Frameworks.Builder.sln the output of bridge.js (and bridge.min.js too) is different:

It skips the header comment of the version, adds instead the header of linqjs.

It adds the BOM between the concatinated files (which makes the compiled script not run, because it the BOM is somewhere in the file, not at the beginning, the javascript engine (v8) recognized it as an illegal token).

And also adds some comments when I compile it on linux/ubuntu

```diff
-    
+    /* Set Methods */
```

Here is the entire output:

```diff
diff --git a/Bridge/Resources/bridge.js b/Bridge/Resources/bridge.js
index c639452..8433875 100755
--- a/Bridge/Resources/bridge.js
+++ b/Bridge/Resources/bridge.js
@@ -1,12 +1,4 @@
-/*
- * @version   : 1.6.0 - Bridge.NET
- * @author    : Object.NET, Inc. http://bridge.net/
- * @date      : 2015-06-23
- * @copyright : Copyright (c) 2008-2015, Object.NET, Inc. (http://object.net/). All rights reserved.
- * @license   : See license.txt and https://github.com/bridgedotnet/Bridge.NET/blob/master/LICENSE.
- */
-
-
+﻿
 // @source Core.js
 
 (function () {
@@ -1334,7 +1326,8 @@
 
     Bridge.String = string;
 })();
-// @source Class.js
+
+﻿// @source Class.js
 
 (function () {
     var initializing = false;
@@ -1732,7 +1725,7 @@
     Bridge.define = Bridge.Class.define;
 })();
 
-// @source Exception.js
+﻿// @source Exception.js
 
 Bridge.define('Bridge.Exception', {
     constructor: function (message, innerException) {
@@ -1955,7 +1948,7 @@ Bridge.define('Bridge.NullReferenceException', {
     }
 });
 
-// @source Interfaces.js
+﻿// @source Interfaces.js
 
 Bridge.define('Bridge.IFormattable', {
     statics: {
@@ -1994,7 +1987,7 @@ Bridge.Class.generic('Bridge.IEquatable$1', function (T) {
 Bridge.define('Bridge.IPromise');
 Bridge.define('Bridge.IDisposable');
 
-// @source Globalization.js
+﻿// @source Globalization.js
 
 Bridge.define("Bridge.DateTimeFormatInfo", {
     inherits: [Bridge.IFormatProvider, Bridge.ICloneable],
@@ -2321,7 +2314,7 @@ Bridge.define("Bridge.CultureInfo", {
     }
 });
 
-// @source Integer.js
+﻿// @source Integer.js
 
 Bridge.define('Bridge.Int', {
     inherits: [Bridge.IComparable, Bridge.IFormattable],
@@ -2848,7 +2841,7 @@ Bridge.define('Bridge.Int', {
 
 Bridge.Class.addExtend(Bridge.Int, [Bridge.IComparable$1(Bridge.Int), Bridge.IEquatable$1(Bridge.Int)]);
 
-// @source Date.js
+﻿// @source Date.js
 
 (function () {
     var date = {
@@ -3625,7 +3618,7 @@ Bridge.Class.addExtend(Bridge.Int, [Bridge.IComparable$1(Bridge.Int), Bridge.IEq
     Bridge.Date = date;
 })();
 
-// @source TimeSpan.js
+﻿// @source TimeSpan.js
 
 Bridge.define('Bridge.TimeSpan', {
     inherits: [Bridge.IComparable],
@@ -3822,7 +3815,7 @@ Bridge.define('Bridge.TimeSpan', {
 
 Bridge.Class.addExtend(Bridge.TimeSpan, [Bridge.IComparable$1(Bridge.TimeSpan), Bridge.IEquatable$1(Bridge.TimeSpan)]);
 
-// @source Text/StringBuilder.js
+﻿// @source Text/StringBuilder.js
 
 Bridge.define('Bridge.Text.StringBuilder', {
     constructor: function () {
@@ -4048,7 +4041,7 @@ Bridge.define('Bridge.Text.StringBuilder', {
         }
     }
 });
-// @source Browser.js
+﻿// @source Browser.js
 
 (function () {
     var check = function (regex) {
@@ -4175,7 +4168,7 @@ Bridge.define('Bridge.Text.StringBuilder', {
     Bridge.Browser = browser;
 })();
 
-// @source /Collections/Interfaces.js
+﻿// @source /Collections/Interfaces.js
 
 Bridge.define('Bridge.IEnumerable');
 Bridge.define('Bridge.IEnumerator');
@@ -4238,7 +4231,7 @@ Bridge.Class.generic('Bridge.IComparer$1', function (T) {
     return Bridge.Class.cache[$$name] || (Bridge.Class.cache[$$name] = Bridge.define($$name, {
     }));
 });
-// @source /Collections/CustomEnumerator.js
+﻿// @source /Collections/CustomEnumerator.js
 
 Bridge.define("Bridge.CustomEnumerator", {
     inherits: [Bridge.IEnumerator],
@@ -4279,7 +4272,7 @@ Bridge.define("Bridge.CustomEnumerator", {
     }
 });
 
-// @source /Collections/ArrayEnumerator.js
+﻿// @source /Collections/ArrayEnumerator.js
 
 Bridge.define('Bridge.ArrayEnumerator', {
     inherits: [Bridge.IEnumerator],
@@ -4315,7 +4308,7 @@ Bridge.define('Bridge.ArrayEnumerable', {
         return new Bridge.ArrayEnumerator(this.array);
     }
 });
-// @source /Collections/Comparer.js
+﻿// @source /Collections/Comparer.js
 
 Bridge.Class.generic('Bridge.EqualityComparer$1', function (T) {
     var $$name = Bridge.Class.genericName('Bridge.EqualityComparer$1', T);
@@ -4365,7 +4358,7 @@ Bridge.Comparer$1.$default = new Bridge.Comparer$1(Object)(function(x, y) {
 
     return Bridge.compare(x, y);
 });
-// @source /Collections/Dictionary.js
+﻿// @source /Collections/Dictionary.js
 
 Bridge.Class.generic('Bridge.KeyValuePair$2', function (TKey, TValue) {
     var $$name = Bridge.Class.genericName('Bridge.KeyValuePair$2', TKey, TValue);
@@ -5203,7 +5196,7 @@ Bridge.define('Bridge.TaskStatus', {
     }
 });
 
-// @source Validation.js
+﻿// @source Validation.js
 
 (function () {
     var validation = {
@@ -5321,11 +5314,11 @@ Bridge.define('Bridge.TaskStatus', {
     Bridge.Validation = validation;
 })();
 
-// @source Attribute.js
+﻿// @source Attribute.js
 
 Bridge.define('Bridge.Attribute');
 
-// @source INotifyPropertyChanged.js
+﻿// @source INotifyPropertyChanged.js
 
 Bridge.define('Bridge.INotifyPropertyChanged');
 
@@ -5335,7 +5328,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
     }
 });
 
-// @source Array.js
+﻿// @source Array.js
 
 (function () {
     var array = {
@@ -5456,7 +5449,14 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
     Bridge.Array = array;
 })();
 
-
+﻿/*--------------------------------------------------------------------------
+ * linq.js - LINQ for JavaScript
+ * ver 3.0.4-Beta5 (Jun. 20th, 2013)
+ *
+ * created and maintained by neuecc <ils@neue.cc>
+ * licensed under MIT License
+ * http://linqjs.codeplex.com/
+ *------------------------------------------------------------------------*/
 
 (function (root, undefined) {
     // ReadOnly Function
@@ -6061,7 +6061,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
 
     // Extension Methods
 
-    
+    /* Projection and Filtering Methods */
 
     // Overload:function(func)
     // Overload:function(func, resultSelector<element>)
@@ -6508,7 +6508,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Join Methods */
 
     // Overload:function (inner, outerKeySelector, innerKeySelector, resultSelector)
     // Overload:function (inner, outerKeySelector, innerKeySelector, resultSelector, compareSelector)
@@ -6582,7 +6582,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Set Methods */
 
     Enumerable.prototype.all = function (predicate) {
         predicate = Utils.createLambda(predicate);
@@ -6998,7 +6998,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Ordering Methods */
 
     Enumerable.prototype.orderBy = function (keySelector, comparer) {
         return new OrderedEnumerable(this, keySelector, comparer, false);
@@ -7093,7 +7093,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Grouping Methods */
 
     // Overload:function(keySelector)
     // Overload:function(keySelector,elementSelector)
@@ -7209,7 +7209,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Aggregate Methods */
 
     // Overload:function(func)
     // Overload:function(seed,func)
@@ -7317,7 +7317,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         return this.sum(selector);
     };
 
-    
+    /* Paging Methods */
 
     Enumerable.prototype.elementAt = function (index) {
         var value;
@@ -7646,7 +7646,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         return result;
     };
 
-    
+    /* Convert Methods */
 
 
     Enumerable.prototype.asEnumerable = function () {
@@ -7735,7 +7735,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
     };
 
 
-    
+    /* Action Methods */
 
     // Overload:function(action<element>)
     // Overload:function(action<element,index>)
@@ -7815,7 +7815,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         }
     };
 
-    
+    /* Functional Methods */
 
     Enumerable.prototype.letBind = function (func) {
         func = Utils.createLambda(func);
@@ -7901,7 +7901,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* Error Handling Methods */
 
     Enumerable.prototype.catchError = function (handler) {
         handler = Utils.createLambda(handler);
@@ -7950,7 +7950,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
         });
     };
 
-    
+    /* For Debug Methods */
 
     // Overload:function()
     // Overload:function(selector)
@@ -8351,7 +8351,7 @@ Bridge.define('Bridge.PropertyChangedEventArgs', {
     Bridge.Linq = {};
     Bridge.Linq.Enumerable = Enumerable;
 })(this);
-
+﻿
 // module export
 if (typeof define === "function" && define.amd) { // AMD
     define("bridge", [], function () { return Bridge; });
```

**Comment by fabriciomurta** [comment]
Yes, on linux the way to merge the scripts are a little different, I must have missed adding the first header. Mind not the minified file, for when bridge outputs files, it is now making the minified file from the non-minified one in real time.

**Comment by fabriciomurta** [comment]
Can you test now if it works better? I couldn't reproduce the problem as not to be able to run the script, but I could find (and remove) the rogue BOM sequences from the `bridge.js` file.

I've fixed it using cygwin + windows, and now am going to try that on OS X as well. Usually linux behaves very close to cygwin, so I am confident it will fix for you. As for OS X... it is another story. :)

**Comment by fabriciomurta** [comment]
Just tested the changes on OS X on cygwin (to see if it broke back) but looks good. Your feedback will be most appreciated.

**Comment by fabriciomurta** [comment]
This was a side effect of fixes in issue #271.
